index.js

```diff
@@ -0,0 +1,125 @@
+        newDiv.remove();
+        cloneChild.append(deleteButton, rightArrowButton);
+        clonedNewDivChild.append(leftArrowButtonDoingSection, deleteButtonDoingSection, rightArrowButtonDoingSection);
+        clonedNewDivChildDone.append(leftArrowButtonDoneSection, deleteButtonDoneSection);
+
+        deleteButton.addEventListener('click', deleteClick);
+        deleteButtonDoingSection.addEventListener('click', deleteClick);
+        deleteButtonDoneSection.addEventListener('click', deleteClick);
+
+        rightArrowButton.addEventListener('click', function(event){
+            doing.append(clonedNewDiv);
+            event.target.closest('.new-div').remove();
+        })
+        leftArrowButtonDoingSection.addEventListener('click', function (){
+            toDo.append(clone);
+            clonedNewDiv.remove();
+        });
+        rightArrowButtonDoingSection.addEventListener('click', function (){
+            done.append(clonedNewDivDone);
+            clonedNewDiv.remove();
+        })
+        leftArrowButtonDoneSection.addEventListener('click', function (){
+            doing.append(clonedNewDiv);
+            clonedNewDivDone.remove();
+        })
+        inputElement.value = ''
+    } else {
+        alert('You have to add a task')
+    }
+}
+function createButtonArea() {
+
+    deleteButton = document.createElement('button');
+    deleteButton.classList.add('delete-button', 'button');
+    const trashIcon = document.createElement('i');
+    trashIcon.classList.add('fa-solid', 'fa-trash-can');
+    deleteButton.append(trashIcon);
+
+    deleteButtonDoingSection = document.createElement('button');
+    deleteButtonDoingSection.classList.add('delete-button-doing', 'button');
+    const trashIconDoingSection = document.createElement('i');
+    trashIconDoingSection.classList.add('fa-solid', 'fa-trash-can');
+    deleteButtonDoingSection.append(trashIconDoingSection);
+
+    deleteButtonDoneSection = document.createElement('button');
+    deleteButtonDoneSection.classList.add('delete-button-done', 'button');
+    const trashIconDoneSection = document.createElement('i');
+    trashIconDoneSection.classList.add('fa-solid', 'fa-trash-can');
+    deleteButtonDoneSection.append(trashIconDoneSection);
+
+    rightArrowButton = document.createElement('button')
+    rightArrowButton.classList.add('arrow', 'button');
+    const rightArrowIcon = document.createElement('i');
+    rightArrowIcon.classList.add('fa-solid', 'fa-arrow-right');
+    rightArrowButton.append(rightArrowIcon);
+
+    rightArrowButtonDoingSection = document.createElement('button')
+    rightArrowButtonDoingSection.classList.add('arrow', 'button');
+    const rightArrowDoingSectionIcon = document.createElement('i');
+    rightArrowDoingSectionIcon.classList.add('fa-solid', 'fa-arrow-right');
+    rightArrowButtonDoingSection.append(rightArrowDoingSectionIcon);
+
+    leftArrowButtonDoingSection = document.createElement('button')
```

Comment:
You forgot to add const on the left side of this line. Because of that, you're creating a global variable.